### PR TITLE
fix(FOROME-794): change the information block of the pie chart

### PR DESCRIPTION
**Pull request by ghost**
https://quantori.atlassian.net/browse/FOROME-794

### CODE REVIEW

**Review by AndreyUstyumenko — APPROVED** [review]

File: src/pages/filter/ui/query-builder/groups/chart/pie-chart/pie-chart.tsx [hunk 1]
```diff
@@ -51,22 +55,30 @@ export const PieChart = ({ data }: IPieChartProps): ReactElement | null => {
           const optionPercentage = ((value / totalCountsOnChart) * 100).toFixed(
             2,
           )
+          const varaintsQuantityContent = t('filter.chart.variants', {
```
Comment (AlMaQntr) [inline src/pages/filter/ui/query-builder/groups/chart/pie-chart/pie-chart.tsx]: I think, we can implement this with pure css flexbox layout. 
Please, try to use `flex-wrap: wrap` and `justify-content: space-between`

File: src/pages/filter/ui/query-builder/groups/chart/pie-chart/pie-chart.tsx [hunk 2]
```diff
@@ -51,22 +55,30 @@ export const PieChart = ({ data }: IPieChartProps): ReactElement | null => {
...
+            value: formatNumber(value),
+          })
+
+          const isLongInfo = (name + varaintsQuantityContent).length > 30
+
+          if (
+            !isLongInfo &&
+            labelsInCollapsedMode !== extendedLabelsInCollapseMode
+          ) {
+            setLabelsInCollapsedMode(extendedLabelsInCollapseMode)
```
Comment (AlMaQntr) [inline src/pages/filter/ui/query-builder/groups/chart/pie-chart/pie-chart.tsx]: It's a bad practice to call `setState` in render flow.

But you still can solve this issue with css only. Try to use flex-box layout in column mode, max-height, flex-wrap and overflow: hidden

**Review by AlMaQntr — CHANGES_REQUESTED** [review]

File: src/pages/filter/ui/query-builder/groups/chart/pie-chart/pie-chart.tsx (hunk 1 above)
Comment (YaroslavTrefilovBraveDevelopers) [reply src/pages/filter/ui/query-builder/groups/chart/pie-chart/pie-chart.tsx]: I changed the implementation

Comment (QYaroslavTrefilov) [reply src/pages/filter/ui/query-builder/groups/chart/pie-chart/pie-chart.tsx]: I changed the implementation

File: src/pages/filter/ui/query-builder/groups/chart/pie-chart/pie-chart.tsx (hunk 2 above)
Comment (QYaroslavTrefilov) [reply src/pages/filter/ui/query-builder/groups/chart/pie-chart/pie-chart.tsx]: Fixed it

**Review by AlMaQntr — APPROVED** [review]

**Pull request merged**